commons/src/main/kotlin/com/simplemobiletools/commons/views/bottomactionmenu/BottomActionMenuPopup.kt

```diff
@@ -68,6 +81,24 @@ class BottomActionMenuPopup(private val activity: BaseSimpleActivity, items: Lis
         }
     }
 
+    private fun adjustUnderlayViewBottomMargin(view: View, showing: Boolean) {
+        bottomActionMenuView.doOnLayout {
+            view.updateLayoutParams {
+                if (this is ViewGroup.MarginLayoutParams) {
+                    val newMargin = if (showing) {
+                        bottomMargin + bottomActionMenuView.height
+                    } else {
+                        bottomMargin - bottomActionMenuView.height
+                    }
+
+                    if (newMargin >= 0) {
+                        bottomMargin = newMargin
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Is this whole height thing added for being able to see the last item in the listview? Because it makes the whole bottom bar higher by default now, I liked the previous height better.

Reply:
Will revert it.